remo/dashboard/views.py

```diff
@@ -170,8 +170,14 @@ def dashboard(request):
         args['reps_without_profile'] = reps.filter(userprofile__registration_complete=False)
 
     if user.groups.filter(Q(name='Admin') | Q(name='Council') | Q(name='Peers')).exists():
-        args['reps_without_mentors'] = reps.filter(
-            userprofile__registration_complete=True, userprofile__mentor=None)
+        # We want to show
+        #   - Reps with completed profiles and no mentor
+        #   - Reps with mentors who are Alumni
+        # while not including any Alumni Reps
+        alumni_ids = User.objects.filter(groups__name='Alumni')
```

Comment:
Sorry my bad when I added the snippet. You need to get the ids for those users:
`alumni_ids = User.objects.filter(groups__name='Alumni').values_list('id', flat=True).

This will return you a list of IDs with that belong to Alumni users.